lesson_02/job1/bll/get_data_from_api.py

```diff
@@ -0,0 +1,23 @@
+import requests
+
+
+AUTH_TOKEN = '2b8d97ce57d401abd89f45b0079d8790edd940e6'
```

Comment:
can not store AUTH_TOKEN in code. Take a look example of python var env https://developer.vonage.com/en/blog/python-environment-variables-a-primer